test/unit/org/tyusupov/TestSpec.groovy

```diff
@@ -9,6 +10,7 @@ import spock.lang.Specification
  * @date 2016-04-28.
  */
 @TestMixin(DomainClassUnitTestMixin)
+@Mock([TestObj, TestObj2])
```

Comment:
Yes, you'll have to mock out each domain object you'll end up using in a unit test.

> The `@Mock` annotation creates mock version of any collaborators. There is an in-memory implementation of GORM that will simulate most interactions with the GORM API.

- https://grails.github.io/grails-doc/latest/guide/testing.html#unitTesting